init-tools.sh

```diff
@@ -109,6 +110,7 @@ if [ ! -e $__INIT_TOOLS_DONE_MARKER ]; then
 
     Tools/crossgen.sh $__scriptpath/Tools
 
+    mkdir -p $__INIT_TOOLS_DONE_MARKER_DIR
     touch $__INIT_TOOLS_DONE_MARKER
```

Comment:
We're just ignoring the failure. Touch fails with exit code 1 and prints the following in the build output.
```
touch: cannot touch '/home/sven/src/dotnet/corefx/Tools/2.0.0-prerelease-01606-01/done': No such file or directory
```
But we don't check the return code so the failure gets ignored.